doc/build-windows.md

```diff
@@ -33,6 +33,7 @@ To get the bash shell, you must first activate the feature in Windows.
 2. Install Ubuntu
   * Open Microsoft Store and search for "Ubuntu 18.04" or use [this link](https://www.microsoft.com/store/productId/9N9TNGVNDL3Q)
```

Comment:
May want to update this and on line 38, to focal/20.04, as the depends system works on that and Bionic is EOL.